terraform/azure/main.tf

```diff
@@ -0,0 +1,174 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "3.52.0"
+    }
+    azuread = {
+      source  = "hashicorp/azuread"
+      version = "2.28.0"
+    }
+  }
+}
+
+provider "azurerm" {
+  features {
+    resource_group {
+      prevent_deletion_if_contains_resources = false
+    }
+  }
+}
+
+data "azurerm_client_config" "current" {}
+
+data "azuread_client_config" "current" {}
+
+resource "azurerm_resource_group" "rg" {
+  name     = var.resource_group_name
+  location = var.location
+}
+
+resource "azurerm_container_registry" "registry" {
+  name                = var.registry_name
+  resource_group_name = azurerm_resource_group.rg.name
+  location            = azurerm_resource_group.rg.location
+  sku                 = "Standard"
+  anonymous_pull_enabled = true
```

Comment:
curious why you need this?

Reply:
Remanence from the copacetic demo. I can remove that. :)

Reply:
Removed that line from the config.